ui/app/com/gu/recipeasy/controllers/Application.scala

```diff
@@ -133,6 +144,10 @@ class Application(override val wsClient: WSClient, override val conf: Configurat
       db.deleteCuratedRecipeByRecipeId(recipeId)
       db.insertCuratedRecipe(curatedRecipeWithId)
       db.moveStatusForward(recipeId)
+      db.getOriginalRecipeStatus(recipeId) match {
```

Comment:
This can be rewritten as:

```
db.getOriginalRecipeStatus(recipeId).map { status =>
   db.insertUserEvent(UserEvent(request.user.email, request.user.firstName, request.user.lastName, recipeId, recipeStatusToUserEventOperationType(status)))
}
```

As it's only performing an operation on the case where we have a status.